Credify/Credify/serviceX.swift

```diff
@@ -166,20 +165,33 @@ public struct serviceX {
         ///   - pushClaimTokensTask: A task that calls your push claim token API. This SDK needs to receive success status of this task.
         ///   - completionHandler: Completion handler. You can get notified about the result of the BNPL flow.
         public func presentModally(from: UIViewController,
-                                   offer: OfferData,
+                                   offers: [OfferData],
```

Comment:
https://github.com/credify-pte-ltd/credify-ios-sdk-kit/pull/4#pullrequestreview-934449915

Reply:
I'm checking it

Reply:
@ngo275  I compiled `getOffers` and `getConnectedProviders` into on method that named with `getOffersAndConnectedProviders `

Reply:
@lqgioan 
`getOffersAndConnectedProviders` is too much information. All marketplaces should know is `getBNPLAvailability`, I would say. Also, the marketplace should know whether BNPL button should be hidden or not. So `getBNPLAvailability` should return a flag that says BNPL is available for users on the marketplaces.

Reply:
@ngo275 I fixed and pushed the code